backend/jupyterlab_metadata_service_server/src/schemas/annotation.js

```diff
@@ -129,6 +134,19 @@ const resolvers = {
         result: newData
       };
     },
+    /**
+     * 
+     */
+    updateAnnotationResolve: async (root, args, { dataSources }) => {
+
+      newData = dataSources.AnnotationAPI.updateResolve(args.annotation);
+
+      return {
+        success: true,
+        message: null,
+        result: newData
```

Comment:
just a comment `updateAnnotationResolve` return a type called `AnnotationReponse` where `result` return a `Annotation`. this function just return `resolved` for result ... it works but if the user try to get the other fields .. the values is `null` and maybe it could be confusing. for now it is OK because maybe we will have changes in a near future.